.github/workflows/automatic-update.yml

```diff
@@ -43,3 +44,9 @@ jobs:
 
             **Notes for reviewer:**
             This is an auto-generated PR which ran `npx -y @grafana/create-plugin@latest update`, updates plugin.json files, grafana-plugin-sdk-go and docker-compose files for each plugin example
+
+      - name: Auto approve PR
+        uses: hmarr/auto-approve-action@v3
+        with:
+          pull-request-number: ${{ steps.create_pr.outputs.pull-request-number }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
shouldn't this be `token: ${{ steps.generate_token.outputs.token }}` as above?

Reply:
no, it should not. Because if we are using the same token it thinks that it approves PR from itself. And it is not possible